mitodl/micromasters
# Update all cache types in update_cache_for_backend

#### What are the relevant tickets?
Related to the recent changes in adding cache for mitxonline backend

#### What's this PR do?
This fixes a bug in the release #5030

#### How should this be manually tested?
Create an account for edx or use an existing account.
When you signup for mitxonline your cache should be fresh and you should not see an error message that data is not fresh on the dashboard.
